optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/main/java/org/optaplanner/workbench/screens/solver/backend/server/SolverValidator.java

```diff
@@ -131,7 +132,7 @@ private ValidationMessage createSolverFactory( final Path resourcePath,
                                                    final KieProject kieWorkbenchProject,
                                                    final boolean runSolver ) {
         final Builder builder = builderCache.assertBuilder( kieWorkbenchProject );
-        final InternalKieModule kieModule = (InternalKieModule) builder.getKieModule();
+        final InternalKieModule kieModule = (InternalKieModule) builder.getKieModuleIgnoringErrors();
```

Comment:
Do you mean that because this is in the SolverValidator, we 'd want to fail fast on errors?